src/components/LocationAutocomplete/LocationAutocomplete.scss

```diff
@@ -1,4 +1,4 @@
-$icon-margin-adjustment: 1px 1px 0px 3px;
+$icon-margin-adjustment: 7.5px 1px 0px 3px;
```

Comment:
I noticed the text "Powered by Google" in the input. Is it something new? I've never seen it before
![Screenshot 2023-07-20 at 11 04 27](https://github.com/textkernel/oneui/assets/19436304/41189b0d-70f2-4c2c-a3a9-24324a3c92be)

Reply:
It is a legal requirement. But it is ok to hide if this component is next to the Map or other places where it is powered by google is present. hence the prop to hide it, but it is there by default (out of my head, so might be wrong)

Reply:
:+1:

Reply:
I'm asking about "Powered by Google" :)

Reply:
No logic for render is here 
![image](https://github.com/textkernel/oneui/assets/104063210/b7fb4e7a-c0f9-459e-a893-1fbff19df290)

Reply:
![image](https://github.com/textkernel/oneui/assets/104063210/f1321eed-6e04-4fe8-9ff0-c4cbf3f0fdba)

Reply:
![image](https://github.com/textkernel/oneui/assets/104063210/b0ba7108-0884-42be-8edd-7b2965116633)

Reply:
ok, so we still can show it. Can we align the "powered by google" text?

Reply:
yes
![image](https://github.com/textkernel/oneui/assets/104063210/45df470c-7c08-409b-ac04-2397e302952b)